### PR TITLE
Fix timeout in the Turn Done tooltip when > 1 hour

### LINKED ISSUE #1718: Turn done time remaining not displaying properly
**Is your feature request related to a problem? Please describe.**
When you hover your mouse over the turn done button on a fixed length turn, it gives you the same information as the button itself.

**Describe the solution you'd like**
Add alternate information about the turn to the popup. Suggest to had a raw time remaining in days:hours:minutes:seconds or Xd / Yh / Zm / As format.

**Describe alternatives you've considered**
A player recalled that this existed before, we might have removed it in a prior commit.

**Additional context**
Discord - https://discord.com/channels/378908274113904641/1061666163651780688/1061666163651780688

### PR DESCRIPTION
**Pull request by lmoureaux**
Closes #1718.

Suggest to backport because the tooltip was doing the wrong thing.

### CODE REVIEW

File: client/turn_done_button.cpp [hunk 1]
```diff
@@ -130,12 +130,38 @@ void turn_done_button::paintEvent(QPaintEvent *event)
...
+    return QString(Q_("?mins/secs:%1min %2s"))
+        .arg(seconds / 60, 2)
+        .arg(seconds % 60, 2);
+  } else if (seconds < 3600) { // < one hour
+    // TRANS: Used in "Time left: 10 minutes". Always at least 5 minutes
+    return QString(_("%1 minutes")).arg(seconds / 60);
+  } else {
+    // Hours and minutes
+    const auto minutes = seconds / 60;
+    return QStringLiteral("%1:%2").arg(minutes / 60).arg(minutes % 60);
```
Comment (jwrober) [inline client/turn_done_button.cpp:155]: Let's add h and m - `QStringLiteral("%1h:%2m)` to follow similar formatting of the others above.

**Review by jwrober — CHANGES_REQUESTED** [review]

**Comment by jwrober** [comment]
Need to rebase after #1745 is done.

File: client/turn_done_button.cpp (hunk 1 above)
Comment (lmoureaux) [reply client/turn_done_button.cpp:155]: `%1h %2min` then?

Comment (jwrober) [reply client/turn_done_button.cpp:155]: yea that is perfect

**Comment by lmoureaux** [comment]
Comment addressed (and branch rebased)

**Review by jwrober — APPROVED** [review]
Perfect.

**Pull request merged**